sys/net/gnrc/sock/include/gnrc_sock_internal.h

```diff
@@ -37,6 +37,22 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default port range for dynamic source port allocation
+ */
+#define GNRC_SOCK_DYN_PORTRANGE_MIN (49152U)
+#define GNRC_SOCK_DYN_PORTRANGE_MAX (65535U)
```

Comment:
I don't understand your last comment. My point was: these are well defined numbers and not GNRC specific, so I would define them in a generic (sic!) header file.